addon/utils/object-transforms.ts

```diff
@@ -0,0 +1,56 @@
+import { isPresent } from '@ember/utils';
+
+type Options = Record<string, string>;
+
+/**
+ * Present config options
+ * @param {Options} opts
+ * @returns {Options}
```

Comment:
It might be prudent to include a reference to `isPresent` in the Ember docs...

```suggestion
 * @returns {Options}
 * @see {@link https://api.emberjs.com/ember/3.28/functions/@ember%2Futils/isPresent|isPresent}
```

Reply:
Sounds good, is that just to give a quick reference to the things that "present" entails here?

Reply:
Yeah, exactly. I always have to double check what "present" really means.